dom/src/DOMSource.ts

```diff
@@ -14,6 +15,7 @@ export interface DOMSource {
   events<K extends keyof HTMLElementEventMap>(
     eventType: K,
     options?: EventsFnOptions,
+    bubbles?: boolean,
```

Comment:
Shouldn't we put `bubbles` inside `options`?

Reply:
I put it extra because it is only used at event listener creation. The options are used later to match individual events or to simulate capture

Reply:
Still, couldn't it be an option? I'm wondering whether this is a public or private API. It's used in the tests, so I'd guess a public API, which is why it should be an option.